.github/workflows/ci.yml

```diff
@@ -41,12 +41,10 @@ jobs:
         conda install -y -c conda-forge xtl xsimd xtensor
 
     - name: Build
-      env:
-        CMAKE_OPTIONS: "\
-          -DHIGHFIVE_PARALLEL_HDF5:BOOL=ON \
-          -DHIGHFIVE_USE_XTENSOR:BOOL=ON"
       shell: bash -l {0}  # required to activate conda
-      run: source $GITHUB_WORKSPACE/.github/build.sh
+      run: |
+        CMAKE_OPTIONS: (-DHIGHFIVE_PARALLEL_HDF5:BOOL=ON -DHIGHFIVE_USE_XTENSOR:BOOL=ON)
+        source $GITHUB_WORKSPACE/.github/build.sh
```

Comment:
Now that this pops up… any particular reason for `source`? I've seen quite a few issues arise when `source` is used to execute other scripts.

Reply:
build is a small script, it's convenient to have it running with the full context, so literally we just move the code to a file on the side

Reply:
Yeah, I was hesitant about `source` but I agree with Fernando that in this context, the build script is so trivial, that this is just fine

Reply:
OK, just making sure :)